source/css/_mixins/base.styl

```diff
@@ -64,15 +64,15 @@ desktop-largest() {
   }
 }
 
-circle() {
+border-radius: 50% {
   border-radius: 50%;
 }
 
-hide() {
+display: none {
   display: none;
 }
 
-show() {
+display: block {
   display: block;
 }
```

Comment:
So, why need that repeated variables?

Reply:
It's funny 😂

Reply:
It looks like I made a mistake with the substitution, but fortunately it didn't cause a syntax error... Stylint doesn't report this either: https://github.com/theme-next/hexo-theme-next/pull/1037

Reply:
Fixed in https://github.com/theme-next/hexo-theme-next/commit/07b05049193c34e58361fb515ed8d8fafa9ddb75